Marlin/ultralcd.cpp

```diff
@@ -848,6 +859,39 @@ void kill_screen(const char* lcd_msg) {
 
   /**
    *
+   * "Driver current control" submenu items
+   *
+   */
+  #ifdef DAC_STEPPER_CURRENT
+  static void dac_driver_getValues() 
+  {
+    driverX = mcp4728_getDrvPct(0);
+    driverY = mcp4728_getDrvPct(1);
+    driverZ = mcp4728_getDrvPct(2);
+    driverE = mcp4728_getDrvPct(3);
+  } 
+
+  static void dac_driver_commit() { mcp4728_setDrvPct(driverX, driverY, driverZ, driverE); }
+
+  static void dac_driver_eeprom_write() { mcp4728_eepromWrite(); }
+  
+  static void lcd_dac_menu()
+  {
```

Comment:
Some points of style:
- Opening brace should be at the end of the previous line
- Indentation is 2 spaces